WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController+FancyAlerts.swift

```diff
@@ -114,13 +114,18 @@ extension BlogDetailsViewController {
         return QuickStartTourGuide.shouldShowChecklist(for: blog)
     }
 
-    @objc func showQuickStartCustomize() {
-        showQuickStart(with: .customize)
-    }
+    @objc func showQuickStart() {
+        let currentCollections = QuickStartToursCollection.collections(for: blog)
+        guard let collectionToShow = currentCollections.first else {
+            return
+        }
+        let checklist = QuickStartChecklistViewController(blog: blog, collection: collectionToShow)
+        let navigationViewController = UINavigationController(rootViewController: checklist)
+        present(navigationViewController, animated: true)
 
-    @objc func showQuickStartGrow() {
-        showQuickStart(with: .grow)
-    }
+        QuickStartTourGuide.shared.visited(.checklist)
+
+        createButtonCoordinator?.hideCreateButtonTooltip()    }
```

Comment:
[nit]
```suggestion
        createButtonCoordinator?.hideCreateButtonTooltip()    
     }
```

Reply:
🙈 
Fixed in a68469b1587ba51363bd7148201d4babf82c0daf